blog/src/__tests__/Featured.test.js

```diff
@@ -0,0 +1,56 @@
+import { render, screen} from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import Featured from '../components/Featured/Featured';
+
+const isProperImageURL = (url) => {
+    const regex = /^https?:\/\/[^\s]+\.(jpg|jpeg|png|gif|webp|avif)$/;
+    return regex.test(url);
+  };
```

Comment:
Kindly write unit tests for this function. Also it would be best to add it to utils folder.